content/contributing/documentation.rst

```diff
@@ -15,407 +15,276 @@ This introductory guide will help you acquire the tools and knowledge you need t
-   documentation:
-
-   .. code-block:: console
+      $ pip install -r requirements.txt
 
-      $ cd documentation/
-      $ pip3 install -r requirements.txt
+   Verify that the installation directory of the Python dependencies is included in your system's
+   `PATH` variable.
 
-.. note::
-   Depending on your :abbr:`OS (Operating System)`, you may need to run the commands ``python`` and
-   ``pip`` instead of ``python3`` and ``pip3``
+   .. tabs::
 
-.. _contributing/make:
+      .. group-tab:: Linux and macOS
 
-Make
-~~~~
+         Follow the `guide to update the PATH variable on Linux and macOS
+         <https://unix.stackexchange.com/a/26059>`_ with the installation path of the Python
+         dependencies (by default :file:`~/.local/bin`).
 
-`Make <https://en.wikipedia.org/wiki/Make_(software)>`_ is a tool that packs a bunch of
-command-lines into one to be easier to remember and to type. In our case, it is used to execute
-complex Sphinx build commands by using a single and simpler one instead.
+      .. group-tab:: Windows
 
-#. Download and install **Make** on your machine.
-#. Verify that `the installation folder of Make is included in your system's PATH variable
-   <win-add-to-path_>`_.
+         Follow the `guide to update the PATH variable on Windows
+         <https://www.howtogeek.com/118594/how-to-edit-your-system-path-for-easy-command-line-access/>`_
+         with the installation path of the Python dependencies.
 
-.. _contributing/pngquant:
+#. Install Make.
 
-pngquant
-~~~~~~~~
+   .. tabs::
 
-`pngquant <https://pngquant.org/>`_ is a tool that we use to compress PNG images so that the
-documentation does not end up weighting several Gigabytes in a few year span.
+      .. group-tab:: Linux and macOS
 
-#. Download and install **pngquant** on your machine.
-#. Verify that `the installation folder of pngquant is included in your system's PATH variable
-   <win-add-to-path_>`_.
+         .. code-block:: console
 
-.. _contributing/prepare-version:
+            $ sudo apt install make -y
 
-Prepare your version
---------------------
+      .. group-tab:: Windows
 
-Now that your machine is all set up, it is time to do the same for your version of the documentation
-files. As it would not be convenient to have several people working on the version {BRANCH} in
-parallel (conflicts of content would occur all the time), and in order to be able to create a
-:abbr:`PR (Pull Request)`, you must `create a new branch
-<https://www.atlassian.com/git/tutorials/using-branches>`_ starting from the branch {BRANCH}. In
-other words, you copy the entirety of this version’s files and give it another name. For this
-example, we will go with ``{BRANCH}-my_contribution``.
+         Follow the `guide to install Make on Windows
+         <https://www.technewstoday.com/install-and-use-make-in-windows>`_.
 
-Execute the following commands in a terminal to...
+#. `Install pngquant <https://pngquant.org/>`_.
```

Comment:
I can't tell for other OS, but for Windows, there is no installer and no explanation on pngquant.org website
my steps were:
- extract the zip file
- cut/paste the `pngquant` folder to C:\Program Files (x86)
- add `C:\Program Files (x86)\pngquant` to the PATH

Reply:
Shame. I will keep it as is for now and maybe come back to it later when I redo a full Windows install, or let other contributors update the steps.

Reply:
Wait, even better :grin: 
![image](https://user-images.githubusercontent.com/15896813/210216279-0146529e-b82e-45fa-9b96-06ce784554cf.png)

Reply:
Hahahhaa ^^